xAPI-Data.md

```diff
@@ -1335,8 +1335,7 @@ outside the scope of this specification.
 ###### Requirements
 
 * The Score Object SHOULD include 'scaled' if a logical percent based score is known.
-* The Score Object SHOULD NOT be used for scores relating to progress or completion.  Consider using an extension
-from an extension profile instead.
+* The Score Object SHOULD NOT be used for scores relating to progress or completion.  Consider using an extension instead.
```

Comment:
I made this exact same change in another PR so this will likely conflict. Not necessarily a big problem, just flagging it up.